README.md

```diff
@@ -107,7 +119,6 @@ git clone git@github.com:AcademySoftwareFoundation/openvdb.git
 cd openvdb
 mkdir build
 cd build
-cmake -DCMAKE_TOOLCHAIN_FILE=<PATH_TO_VCPKG>\scripts\buildsystems\vcpkg.cmake ..
-make -j4
-make install
```

Comment:
If we change one instance of `make -j4` to `cmake --build . --parallel 4` we should change them all IMO.

Unless there's a good reason for using the cmake wrapper, I'd prefer using make directly for simplicity.

Reply:
Well make doesn't work out the box on Windows so you have to use cmake to build

Reply:
Right I see, that's a good enough reason. Could we just update the other references in this documentation to use `cmake --build` too?

Perhaps the `--config Release` part could use CMAKE_BUILD_TYPE in the line above instead?

Reply:
>  Could we just update the other references in this documentation to use cmake --build too?

I think we need a dedicated "Building on Windows" separate from the UNIX instructions. Both the dependency and build pages need update. I'll look to doing these separately after I've work through all the build components.

> Perhaps the --config Release part could use CMAKE_BUILD_TYPE in the line above instead?

Actually this only for multi tool configurations which we don't currently support (you choose the build type when you initially run cmake) so I can remove this.

Reply:
> > Perhaps the --config Release part could use CMAKE_BUILD_TYPE in the line above instead?
> 
> Actually this only for multi tool configurations which we don't currently support (you choose the build type when you initially run cmake) so I can remove this.

Okay this isn't true, without `--config Release` CMake will try to build both release and debug versions of the library and these instructions are specifically for Release - so it needs to be `--config Release`